src/components/buttons/СategoryButton/CategoryButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import { TouchableOpacity, Text } from 'react-native';
+import { useStyles } from 'react-native-unistyles';
+import { SvgProps } from 'react-native-svg';
+import { useTranslation } from 'react-i18next';
+
+import { styleSheet } from './CategoryButton.style';
+import { Categorys } from './CategoryButton.constans';
+import { sizes } from '@src/styling/sizes';
+
+type CategoryButtonProps = {
+	category: Categorys;
+	SvgIcon: React.FC<SvgProps>;
+	onPress: (category: Categorys) => void;
+};
+
+export const CategoryButton = ({ category, SvgIcon, onPress }: CategoryButtonProps) => {
+	const [isActive, setIsActive] = useState<boolean>(false);
+	const { t } = useTranslation();
+	const { styles } = useStyles(styleSheet);
+
+	const handleOnPres = () => {
+		setIsActive(!isActive);
+		onPress(category);
```

Comment:
This code only proves my theory. `const [isActive, setIsActive] = useState<boolean>(false);` is a. redundant state